mypy.ini

```diff
@@ -139,6 +139,9 @@ ignore_errors = True
 [mypy-torch.nn.parallel._functions]
 ignore_errors = True
 
+[mypy-torch.nn.parallel.comm]
```

Comment:
Add the torch/nn/parallel/comm.py (from torch/cuda/comm.py) to white list.

The required attribute _broadcast, _broadcast_coalesced, _scatter and _gather are statically registered by the device runtime.